Source/m_menu.c

```diff
@@ -1371,7 +1371,7 @@ menuitem_t MouseMenu[]=
   {-1,"",0},
   {2,"M_VERSEN",M_MouseVert,'v', "VERTICAL"},
   {-1,"",0},
-  {2,"",M_ControllerTurn,'g',"GAMEPAD"},
+  {2,"M_PADSEN",M_ControllerTurn,'g',"GAMEPAD"},
```

Comment:
Is this the lump name used in other ports or just a dummy?

Reply:
GZDoom and Eternity do not use graphic lumps for the gamepad menu, I have not checked other ports.